tracer/src/Datadog.Trace/RemoteConfigurationManagement/Transport/RemoteConfigurationApi.cs

```diff
@@ -55,6 +55,7 @@ public async Task<GetRcmResponse> GetConfigs(GetRcmRequest request)
             var apiRequest = _apiRequestFactory.Create(uri);
 
             var requestContent = JsonConvert.SerializeObject(request);
+            Log.Debug("Sending Remote Configuration Request: {0}", requestContent);
```

Comment:
nit: don't use `{0}` in log format strings (though it doesn't _actually_ make much difference for our use case!)
```suggestion
            Log.Debug("Sending Remote Configuration Request: {Content}", requestContent);
```

Reply:
Agreed, that's a nice touch! Fixed in 8783284f81fabc53121b1c1a2d37507487718b80